test/studies/champs/functions.bash

```diff
@@ -60,7 +60,7 @@ function test_compile() {
   local kind=$@
 
   test_start "make $kind"
-  make release MOD=$kind 2> $kind.comp.out.tmp
+  make release MOD=$kind NPROC=0 2> $kind.comp.out.tmp
```

Comment:
In practice won't this just end up invoking a single call to `chpl` so we wouldn't get any parallelism anyway?

I suppose it adds some wasted overhead, is that your motivation for forcing NPROC=0?

Reply:
Ah, I understand the confusion.

There's always a single call to `chpl`. `NPROCS` is an internal Makefile flag in CHAMPS that controls whether `chpl` should launch with `-j` flag. Without an explicit `NPROCS`, CHAMPS will effectively do `chpl -j $(nprocs)`. You need to set it to 0 to prevent `-j`.

(`chpl -j` will use sequential front end, but will compile the generated C code in parallel using `make -j` under the hood. This is good for compilation speed and memory footprint, but not for performance. See https://chapel-lang.org/perf/champs/16-node-cs-hdr/?configs=cbackend&graphs=buildtime,flowsimulationbreakdown)